src/module-server/middleware/js.ts

```diff
@@ -67,30 +67,30 @@ export const jsMiddleware = async ({
     try {
       // Normalized path starting with slash
       const path = posix.normalize(req.path);
+      const params = new URLSearchParams(req.query as Record<string, string>);
```

Comment:
I wanted to flag the warnings above. They don't seem critical and I don't see them as a merge blocker but wanted to make sure you were aware.

(For some reason GitHub wouldn't let me comment directly on them.

Reply:
Thanks 👍

I'm not sure whether at this point there is a good opportunity to factor out part of this function. For now I think I'll leave it as-is but I'll keep my eyes out for refactoring opportunities 👀

Reply:
Cool, works for me :+1: